src/main/java/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java

```diff
@@ -246,5 +247,10 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(247, "GEO-Miner", 24), SlimefunItems.GEO_MINER);
 	    Slimefun.registerResearch(new Research(248, "Lightning Rune", 24), SlimefunItems.RUNE_LIGHTNING);
 	    Slimefun.registerResearch(new Research(249, "Totem of Undying", 36), SlimefunItems.TOTEM_OF_UNDYING);
+		Slimefun.registerResearch(new Research(250, "Pork Jerky", 2), SlimefunItems.PORK_JERKY);
+		Slimefun.registerResearch(new Research(251, "Chicken Jerky", 2), SlimefunItems.CHICKEN_JERKY);
+		Slimefun.registerResearch(new Research(252, "Mutton Jerky", 2), SlimefunItems.MUTTON_JERKY);
+		Slimefun.registerResearch(new Research(253, "Rabbit Jerky", 2), SlimefunItems.RABBIT_JERKY);
+		Slimefun.registerResearch(new Research(254, "Fish Jerky", 2), SlimefunItems.FISH_JERKY);
```

Comment:
Bad indentation.
You could also argue that all the Jerky variants could belong to the same research.